amplify.yml

```diff
@@ -11,12 +11,12 @@ frontend:
         - yarn install
     build:
       commands:
-        - REACT_APP_APES_AUTH_SECRET_KEY=${REACT_APP_APES_AUTH_SECRET_KEY}
-        - REACT_APP_ANGEL_AUTH_SECRET_KEY=${REACT_APP_ANGEL_AUTH_SECRET_KEY}
```

Comment:
`REACT_APP_APES_AUTH_SECRET_KEY` & `REACT_APP_ANGEL_AUTH_SECRET_KEY` should no longer be needed since AWS handles this via domain checks and CORS on the API gateway. cc: @jp-mariano

Reply:
i tried to remove these secrets but queries are failing e.g. marketplace cards doesn't load

Reply:
From Localhost or something else? I don't think we only have allowlisted localhost on port 4200 domains setup in the API Gateway.

Reply:
removed from latest update in RC-v1.9